samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_eu.properties

```diff
@@ -85,6 +85,7 @@ loading=Galdera kargatzen ari da...
 # CALCULATED_QUESTION
 calculated_question=Galdera kalkulatua 
 calc_question_var_label=Aldagaiak
+calc_question_globalvar_label=Aldagai globala
```

Comment:
All the new variables at the end of the file with a comment.

Reply:
Added at the end.